SIM-ONE Training/simone_training/models/base.py

```
@@ -227,21 +277,132 @@ def __init__(
+        hidden_states = self.norm(hidden_states)
+        logits = self.lm_head(hidden_states)
+
+        return logits, hidden_states, last_attn_weights
+
+    def forward(
+        self,
+        input_ids: torch.Tensor,
+        attention_mask: Optional[torch.Tensor] = None,
+        output_governance: bool = False,
+        prophetic_state: Optional[PropheticSingularityState] = None,
+        **kwargs,
```

Ruff (0.13.1)

Unused method argument: `kwargs`

(ARG002)